handbook/brand.md

```diff
@@ -207,22 +207,22 @@ In Figma:
   * Avoid using SVGs or icon fonts.
 3. Click the __Export__ button.
 
-## Brand rituals
+## Rituals
 
-The following rituals are engaged in by the  directly responsible individual (DRI) and at the frequency specified for the ritual.
+Directly Responsible Individuals (DRI) engage in the rituals below at the frequency specified.
```

Comment:
Yours is better, but I think we can still make this more friendly. Something like:
"The following table lists the Brand group's rituals, frequency, and directly responsible individual (DRI)."  

That would need to be consistent across all groups, swapping out "Brand," obvs.

Reply:
I like it.